### DISCUSSION TITLE
Reinforcement activitytest

### DISCUSSION THREAD

**Discussion by Multri** [rettichlp/UnicacityAddon-1.12.2#363] [Ideas]
Bitte den screenshot von activitytest eine halbe sekunde oder so verzögert machen, oder eine option hinzufügen dass man das eine sekunde verzögert einstellen kann. Cop  Leader wollen bei den screens jetzt sehen dass man unterwegs ist

**Comment by rettichlp** [comment]
Wir sind dran, aber das ist etwas schwieriger als gedacht 👍🏻

**Reply by rettichlp** [reply]
Siehe #306

**Reply by Multri** [reply]
vielen dank :)
